LICENSE

```diff
@@ -1,23 +1,21 @@
-# The MIT License (MIT)
+The MIT License (MIT)
 
-Copyright (c) 2016 Thane Thomson
+Copyright (c) 2023 
```

Comment:
```suggestion
Copyright (c) 2016- Thane Thomson and contributors
```

The start date from which copyright is valid should be indicated - not the current date. Also, afaik, most countries recognize that copyright of each individual contribution belongs to the specific contributor.

Reply:
`flit init` did that automatically, thanks for noticing it :+1: